src/playbackstrategy/basicstrategy.js

```diff
@@ -36,14 +37,16 @@ function BasicStrategy(mediaSources, windowType, mediaKind, playbackElement) {
   }
 
   function load(_mimeType, startTime) {
-    if (!mediaElement) {
+    if (mediaElement == null) {
       setUpMediaElement(startTime)
       setUpMediaListeners()
-    } else {
-      mediaElement.src = mediaSources.currentSource()
-      setStartTime(startTime)
-      mediaElement.load()
+
+      return
     }
+
+    mediaElement.src = mediaSources.currentSource()
+    setStartTime(startTime)
+    mediaElement.load()
```

Comment:
If we return in the else, can't lines 47 to 49 be inside the if, and then if that's true, can't we just eliminate the else entirely?